### ISSUE TITLE
accuracy is always 1.00

### ISSUE THREAD

**Issue by hjian42** [sarweshsuman/keras-sentiment-analytics#1]
when I train the model using yours or my data, the accuracy is always 1.00 after several epochs. I find it a bit weird because sentiment analysis should not have such high accuracy...
